app/Insightly/Listeners/CreateOpportunity.php

```diff
@@ -19,18 +19,14 @@ final class CreateOpportunity implements ShouldQueue
     use Queueable;
 
     public function __construct(
-        private readonly InsightlyClient $insightlyClient,
-        private readonly IntegrationRepository $integrationRepository,
-        private readonly InsightlyMappingRepository $insightlyMappingRepository
+        private readonly InsightlyClient               $insightlyClient,
+        private readonly EloquentIntegrationRepository $integrationRepository,
```

Comment:
Maybe also the interface is required here? This function also has the same auto-indentation from PHPStorm

Reply:
See https://github.com/cultuurnet/publiq-platform/pull/28#discussion_r1045616888